### PR TITLE
Tic Tac Toe Game

### PR DESCRIPTION
**Pull request by RazaAli313**
## PR Description 📜

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
Fixes # < 3173>
Issue Number #3173

<hr>
 
## Mark the task you have completed ✅
I have built # Tic Tac Toe
# Description 📃
This is a classic Tic Tac Toe game implemented with additional functionalities such as success music on winning, music on game drawn, and sound on each click of turn. The game features a user-friendly interface and supports both single-player and two-player modes.

## Functionalities 🎮
- [x] Play against another player
- [x] Success music on winning
- [x] Music on game drawn
- [x] Sound on each click of turn
- [x] Recognizes winning combinations horizontally, vertically, and diagonally
- [x] Attractive UI

## How to Play? 🕹️
1. Launch the game.
2. Click on the desired cell to place your symbol (X or O).
3. Alternate turns with your opponent.
4. The first player to achieve three of their symbols in a row wins.
5. If all cells are filled and no player has three symbols in a row, the game is drawn.

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @RazaAli313 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by RazaAli313** [comment]
@kunjgit Sir kindly review My PR

File: README.md [hunk 1]
```diff
@@ -185,7 +185,8 @@ This repository also provides one such platforms where contributers come over an
...
+| [Emoji_Intruder](https://github.com/kunjgit/GameZone/tree/main/Games/Emoji_Intruder)                             | [Guess The Weapon](https://github.com/kunjgit/GameZone/tree/main/Games/Guess_The_Weapon)                | [Guess Who](https://github.com/kunjgit/GameZone/tree/main/Games/Guess_Who)          
+Tic Tac Toe(https://github.com/RazaAli313/GameZone/tree/main/Games/Tic_Tac_Toe_Game)|                                                                                                                 |                                                                                                             |
```
Comment (ishita-43) [inline README.md:189]: follow the format while entering your game

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ishita-43) [reply README.md:189]: Instead of your repository name update this with kunjgit to makke it workable on the website

Comment (ishita-43) [reply README.md]: Manually resolved will now be merged

**Review by ishita-43 — APPROVED** [review]
Ready to merge

**Comment by kunjgit** [comment]
Thank you  @RazaAli313 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**